### PR TITLE
RT-1.2: OTG - Setting DefaultNetworkInstance on the device

### PR DESCRIPTION
**Pull request by stthummuri**
Network instance type is set to "DEFAULT".
It is mandatory for user to specify type of network instance. For DEFAULT network instance type is "DEFAULT"

Porting changes from #624 to OTG

### CODE REVIEW

File: feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go [hunk 1]
```diff
@@ -528,6 +528,11 @@ func TestEstablish(t *testing.T) {
 	t.Logf("Start DUT interface Config")
 	configureDUT(t, dut)
 
+	// Configure Network instance type on DUT
```
Comment (greg-dennis) [inline feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go:531]: Should be a period at the end. That's supposed to be true throughout, although I see it's not quite today

**Review by greg-dennis — APPROVED** [review]
One small comment

File: feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go [hunk 2]
```diff
@@ -528,6 +528,11 @@ func TestEstablish(t *testing.T) {
...
+	t.Log("Configure Network Instance type")
+	dutConfNIPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
+	gnmi.Replace(t, dut, dutConfNIPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+
```
Comment (sachendras) [inline feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go:535]: Why is this section different between ATE and OTG versions of the script. Below from the ATE

// Configure Network instance type on DUT
	t.Log("Configure Network Instance type")
	dutConfNIPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
	dutConfNIPath.Type().Replace(t, telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)

File: feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go (hunk 1 above)
Comment (stthummuri) [reply feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go]: Done.

**Review by greg-dennis — APPROVED** [review]
Re-approving for Go quality.

**Pull request merged**